java/format/src/main/java/module-info.java

```diff
@@ -15,6 +15,7 @@
  * limitations under the License.
  */
 
+@SuppressWarnings({ "requires-automatic", "requires-transitive-automatic" })
```

Comment:
@lidavidm is this acceptable?

Reply:
I thought we upgraded flatbuffers to avoid this?

Reply:
@danepitkin we did an update recently?

Reply:
We created a PR in flatbuffers, but it hasn't been reviewed yet. https://github.com/google/flatbuffers/pull/8367

Reply:
Typical Google project. Well, that means this sorta defeats the purpose of modularization in the first place, but I suppose there's no alternative.

Reply:
Let's link to the Flatbuffers issue and clearly note that it's Google's problem here

Reply:
Sure, I will do that.

Reply:
Added a comment by linking https://github.com/apache/arrow/issues/44037